docs/api/components/schemas.yaml

```diff
@@ -2317,6 +2317,10 @@ Log-rotation-config-readonly:
       description: Max Size (in mb) of log files before deletion
       type: integer
       default: 1024
+    rotation_interval:
+      description: If set, the interval at which log files are rotated, even if max_size is not reached.
```

Comment:
I would add that these are units of hours?

Reply:
It's a duration string so can take any Go duration unit. I'll copy/paste the units description from `min_config_fetch_interval`